spec/sections/standard.adoc

```diff
@@ -362,22 +363,28 @@ default value MUST be treated as false. If true, the `command` placeholder will
 containing all files. If false or omitted, the `command` placeholder will be replaced with an absolute path to a single
 file.
 
+|`partial`
+|Optional
+2+|MUST be a boolean indicating whether input file is required in whole or in part. This allows an executor† to make
+intelligent choices when providing very large files to a job. This should only be set to true if the job is expected to
+use less than half of very large input files. If omitted, the default value MUST be treated as true.
```

Comment:
Shouldn't the default for partial be false?

Reply:
+

Reply:
Yikes, good catch.